### PR TITLE
fix(tooltip): pointer-events disabled on button toggles

### PR DESCRIPTION
**Pull request by denkristoffer**
A fix for #805

See https://github.com/facebook/react/issues/4251#issuecomment-267004045

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -200,7 +200,7 @@ export const Tooltip = ({
     <>
       <ContainerElement
         ref={elementRef}
-        className={targetWrapperClassName}
+        className={cn(styles.TooltipContainer, targetWrapperClassName)}
```
Comment (nikazawila) [inline packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx:203]: question or a possible nitpick - do we use camelCase or CapitalCamelCase in f36?

Comment (denkristoffer) [reply packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx:203]: With regards to CSS we seem to use a mix, of course 🙃😂 Almost all of it is like this though. I've always known it as PascalCase 🙂

Comment (nikazawila) [reply packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx:203]: 🙃
I'd guess we could fix it then in the whole file so it's consistent at least on the 'per file' basis?

Comment (gui-santos) [reply packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx:203]: I think the PascalCase one is a class from the css file, not sure why we are using PascalCase to name the classes but I think every CSS class is using that pattern

And `targetWrapperClassName` is a prop and we are always using camelCase for props (this one makes sense for me hahahaha)

One good news is that we will move all styles to emotion so we could make everything camelCase then 🙌

**Review by gui-santos — APPROVED** [review]

File: packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (denkristoffer) [reply packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx:203]: Yeah, CSS file class names are PascalCase, everything else is camelCase. However, I've seen camelCase names from CSS in a few files as well 😅

**Pull request merged**